process/mock/topicHandlerStub.go

```diff
@@ -26,6 +27,11 @@ func (ths *TopicHandlerStub) RegisterMessageProcessor(topic string, handler p2p.
 	return ths.RegisterMessageProcessorCalled(topic, handler)
 }
 
+// ID -
+func (ths *TopicHandlerStub) ID() core.PeerID {
+	return "peer ID"
```

Comment:
You can add IDCalled method and call it here if it is not nil

Reply:
done